src/agent/coverage/src/cobertura.rs

```diff
@@ -195,20 +244,22 @@ mod tests {
 
         _emitter_test.write(
             XmlEvent::start_element("line")
-                .attr("number", "0")
+                .attr("number", "1")
                 .attr("hits", "0")
                 .attr("branch", "false"),
         )?;
-        _emitter_test.write(XmlEvent::end_element())?; // line
 
+        _emitter_test.write(XmlEvent::end_element())?; // line
         _emitter_test.write(XmlEvent::end_element())?; // lines
         _emitter_test.write(XmlEvent::end_element())?; // class
         _emitter_test.write(XmlEvent::end_element())?; // classes
         _emitter_test.write(XmlEvent::end_element())?; // package
         _emitter_test.write(XmlEvent::end_element())?; // packages
         _emitter_test.write(XmlEvent::end_element())?; // coverage
 
+        //println!("{}", source_coverage_result?);
```

Comment:
We should be sure to prune stray / commented dev code, like here and below.

Reply:
Opps, done!